src/app/movies/movies.component.ts

```diff
@@ -36,6 +36,7 @@ export class MoviesComponent implements OnInit, OnDestroy {
   // }
   addMovie(): void {
     this.movies.unshift(this.movie);
```

Comment:
w tym komponencie w ogóle nie powinno być modyfikowalnej tabeli movies, także ten unshift jest błędem. W addMovie powinieneś jedynie: 1. Przerobić input usera na obiekt filmu 2. Wywołać przygotowaną akcję ngrx (przez fasadę lub od bidy bezpośrednio przez this.store.dispatch(fromMovies.addMovie(movie)) (ale możesz poprzestać obecnie na console.logu i dodac store później, to jest ok)